plugin/core/settings.py

```diff
@@ -187,7 +187,7 @@ def read_client_config(name: str, client_config: Dict) -> ClientConfig:
         client_config.get("env", dict()),
         client_config.get("tcp_host", None),
         client_config.get("tcp_mode", None),
-        client_config.get("experimental_capabilities", config.experimental_capabilities)
+        client_config.get("experimental_capabilities", dict())
```

Comment:
Should this be `None` instead of `dict()`? We're checking for `is not None` in `get_initialize_params`.

Reply:
In theory yes but then `initializationOptions` should be None as well [above](https://github.com/sublimelsp/LSP/pull/971/files#diff-c7b7ba6d8b0930a59af07dc6cdb03159R185) ?

Reply:
I mean we also do `if config.init_options is not None` in `get_initialize_params` and `init_options` has `dict()` as default value in `read_client_config`. Should they then have the same default ?

Reply:
Good observation. Well, I guess it doesn't really matter for this PR.